scripts/review-bot-prs.py

```diff
@@ -1,15 +1,37 @@
 #!/usr/bin/env python
 """
-Show a human all Dependabot PRs that look ready to be merged. If the human accepts, approve and merge the PRs.
+Show a human bot PRs that look ready to be merged. If the human accepts, approve and merge the PRs.
 
 Requires GitHub's CLI tool: https://github.com/cli/cli
+
+Usage:
+    scripts/review-bot-prs.py <bot>
+
+Available bots:
+    - dependabot
+    - snyk
 """
 
+import sys
 import json
 import subprocess
+from docopt import docopt
+
+sys.path.insert(0, '.')
```

Comment:
Is this needed? I think we only need to add this when we're importing from somewhere else within this repo.